metals/src/main/scala/scala/meta/internal/metals/JvmOpts.scala

```diff
@@ -14,21 +14,39 @@ import scala.meta.io.AbsolutePath
  */
 object JvmOpts {
 
-  def fromWorkspaceOrEnv(workspace: AbsolutePath): List[String] = {
-    val jvmOpts = workspace.resolve(".jvmopts")
-    if (jvmOpts.isFile && Files.isReadable(jvmOpts.toNIO)) {
-      val text = FileIO.slurp(jvmOpts, StandardCharsets.UTF_8)
-      text.linesIterator.map(_.trim).filter(_.startsWith("-")).toList
+  /**
+   * Tries to get jvmopts needed for running tests, which is now not possible to do with test explorer.
+   * It will also try to use .jvmopts, filter out any -X options since they proved to be problematic,
+   */
+  def fromWorkspaceOrEnvForTest(workspace: AbsolutePath): List[String] = {
+    val forTests =
+      fromWorkspaceOrEnv(workspace, ".test-jvmopts", "TEST_JVM_OPTS")
```

Comment:
Looking at this now, I think that maybe empty `.test-jvmopts` should override `.jvmopts`.

Reply:
Right, I wanted to avoid using an Option, but it makes sense here. Changed the signatures, should work the way you describe